Documentation/Blazorise.Docs/Components/ComponentApiDocs.razor

```diff
@@ -0,0 +1,48 @@
+﻿@using Blazorise.Generator.Features.ApiDocsDtos
+<DocsPageSubtitle>
+    API
+</DocsPageSubtitle>
+
+@{
+    IReadOnlyList<ApiDocsForComponent> apiDocsForComponents = ComponentTypes.Select(x => ComponentsApiDocsSource.Instance.Components.GetValueOrDefault(x)).Where(x => x is not null).ToList();
+    var hasComponentTypes = ComponentTypes.Any();
+    var multipleComponentTypes = ComponentTypes.Count > 1;
+    var hasMethods = apiDocsForComponents.Any(x => x.Methods.Any());
+}
+
+@if (hasComponentTypes)
+{
+    <ComponentApiDocsProperty ApiDocsForComponents="apiDocsForComponents"
+                              MultipleComponentTypes="multipleComponentTypes"
+                              PropertyType="ComponentApiDocsProperty.PropertyTypes.Parameters"/>
+
+
+    <ComponentApiDocsProperty ApiDocsForComponents="apiDocsForComponents"
+                              MultipleComponentTypes="multipleComponentTypes"
+                              PropertyType="ComponentApiDocsProperty.PropertyTypes.Events"/>
+}
+
+@if (hasMethods)
```

Comment:
I'd just do an enumerable with the methods straight away calculated **oninit** and check whether it's not empty and proceed with that single collection, instead of having hasMethod + enumerable.

i.e
@if(!apiDocsForComponentsMethods.IsNullOrEmpty())
...
foreach (var in apiDocsForComponentsMethods...)